packages/ui/react-ui/src/components/Buttons/IconButton.tsx

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import React from 'react';
+
+import { Button, type ButtonProps } from './Button';
+import { useThemeContext } from '../../hooks';
+import { Icon, type IconProps } from '../Icon';
+
+type IconButtonProps = Omit<ButtonProps, 'children'> & Pick<IconProps, 'icon'> & { label?: string };
+
+const IconButton = ({ icon, label, ...props }: IconButtonProps) => {
+  const { tx } = useThemeContext();
+  return (
+    <Button {...props} classNames={tx('iconButton.root', 'iconButton')}>
+      <Icon icon={icon} />
+      {label && <span>{label}</span>}
```

Comment:
```suggestion
      {label && <span className='sr-only'>{label}</span>}
```
I assume this is what you meant to use?

Reply:
but this would make it invisible?

Reply:
If this should be an option, perhaps provide that as a flag property of `IconButton`

Reply:
> the intention is in the image above
But I see about as many instances of icon-only buttons, and based on the name of the component I’d assumed it would have that capability. You gave this moose a muffin and he is asking for some jam 😅